jquery.sumoselect.js

```diff
@@ -483,8 +483,8 @@
                     O.placeholder = O.placeholder ? (settings.prefix + ' ' + O.placeholder) : settings.placeholder
 
                     //set display text
-                    O.caption.html(O.placeholder);
-                    O.CaptionCont.attr('title', O.placeholder);
+                    O.caption.text(O.placeholder);
+                    O.CaptionCont.attr('title', $('<span></span>').text(O.placeholder).html());

```

Comment:
This change to try escape the title attribute is unnecessary and caused the following
![untitled9](https://user-images.githubusercontent.com/468816/44030454-81d035e0-9f09-11e8-912a-32b687ce28ff.png)

Otherwise the PR looks good and I hope it would be merged asap.